WordPress/gutenberg
# Alt image setting

## Description
This PR implements the Alt settings on image blocks.
The UI is temporary, so don't worry about the Textarea control being in the bottom-sheet itself, or the styles of it. We are working on it with Thomas.

Also: 
- Imported `BaseControl` and `TextareaControl` for parity with Web.
- Removed the `Done` button from the bottom-sheet as requested by Thomas.
- Small tweaks on the BottomSheet.Cell to behave better with longer texts.
- Made the bottom-sheet avoid the keyboard on iOS. (On Android it was already working)

**(Updated GIF)**
![alt-setting](https://user-images.githubusercontent.com/9772967/52134797-b0f52c00-2644-11e9-9d9b-a61d24774ac9.gif)


To test:
- Run the example project from the related `gutenberg-mobile` branch (https://github.com/wordpress-mobile/gutenberg-mobile/pull/531)
- Select an image block with image.
- Tap on the settings button (wheel in the inline tool bar)
- In the bottom-sheet, write something in the textinput control.
- Check that the Alt change.
- Close the bottom-sheet (swipe down or tap in the background)
- Open it again.
- Check that the old Alt value is there.
